src/main/java/slimeknights/tconstruct/common/TinkerOredict.java

```diff
@@ -70,6 +72,24 @@
 public class TinkerOredict {
 
   public static final String PulseId = "TinkerOredict";
+  public static final String[] dyes = { // makes oredict to int a bit easier in a couple other places
```

Comment:
Yes, but it is a local variable, [see here]https://github.com/MinecraftForge/MinecraftForge/blob/master/src/main/java/net/minecraftforge/oredict/OreDictionary.java#L252)